### DISCUSSION TITLE
Colorize tabs - change colors

### DISCUSSION THREAD

**Discussion by pyinto** [mbnuqw/sidebery#1688] [General]
Trying to change my tab's color in Sidebery since they aren't populated from Firefox Color extension's theme. I found this option:
_Inherit custom (manually set) color in the child tabs_
but don't really see where those colors are set? Would be great if somebody could point out exactly how and where those colors are located, so I can change styles manually.
I understand that color is set using [tab-color](https://github.com/mbnuqw/sidebery/blob/aa476a47f3663230ee5722d1a3cebffe4faf3192/src/styles/themes/proton/sidebar/tab.styl#L247) property, but can't get exactly from where those default colors are extracted.

Thanks!

**Comment by colrobloxkid69420** [comment]
+1

**Comment by ahahahaah** [comment]
+2

**Comment by mbnuqw** [comment]
Custom (manually set) color is a color set via context menu (if you don't have it, check context menu editor)

**Reply by ahahahaah** [reply]
my concern was about colors of   all new tabs, and the color of the active tab.
i wanted to have the color of active tab very different than the color of the other tabs.
how to do it?
thanks
